driver/CMakeLists.txt

```diff
@@ -142,10 +152,6 @@ target_link_libraries (${libname}-impl
 
 add_library (${libname} SHARED
     api/odbc.cpp
```

Comment:
Kinda unrelated, but why `odbc.cpp` is not in `${libname}-impl` target, like the rest?

Reply:
`odbc.cpp` doesn't contain anything that could be unit tested, for example. The ultimate plan is to remove any logic from `odbc.cpp` leaving only API function trivial definitions in there.